ui/revenuecatui/build.gradle

```diff
@@ -24,6 +25,9 @@ android {
 
     defaultConfig {
         minSdkVersion 24 // MeasureFormat requires API 24
+        vectorDrawables {
+            useSupportLibrary true
+        }
```

Comment:
This was added by the wizard when I created the activity. Need to test whether it's actually needed